draft-ietf-quic-multipath.md

```diff
@@ -1368,7 +1366,7 @@ the "QUIC Transport Parameters" registry under the "QUIC Protocol" heading.
 
 Value                                         | Parameter Name.   | Specification
 ----------------------------------------------|-------------------|-----------------
-TBD (current version uses 0x0f739bbc1b666d07) | initial_max_paths | {{nego}}
+TBD (current version uses 0x0f739bbc1b666d07) | initial_max_path_id | {{nego}}
 {: #transport-parameters title="Addition to QUIC Transport Parameters Entries"}
```

Comment:
I think we should change the parameter ID. The previous value meant "max path = 1 means path_id < 1"; we are changing that to "max path = 1 means path_id <= 1". That's an incompatible change, thus we need a new parameter ID value.

Reply:
It's true. I've changed the parameter ID from 0x0f739bbc1b666d07 to 0x0f739bbc1b666d09.